rof-scheduler/src/SharedServices/inputValidationService.js

```diff
@@ -219,3 +225,55 @@ export const ensureCreateClientInformationProvided = function (
 
   return validationErrors;
 };
+
+export const ensureClientUpdateInformationProvided = function (
```

Comment:
This is fine for now. I realized that our account settings page does not work for clients.

We will need to change that later and then we can combine the two updateInformation checks.